### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by cedricbousmanne**

### CODE REVIEW

**Comment by guguso** [comment]
:+: Can we add this in the project?

**Comment by tarraschk** [comment]
👍

File: lib/will_paginate/locale/fr.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    next_label: "Suivant &#8594;"
+    page_gap: "&hellip;"
+
+    page_entries_info:
+      single_page:
+        zero:  "Aucun %{model} trouvé"
+        one:   "Affichage de 1 %{model}"
+        other: "Affichage de tout les %{count} %{model}"
+      single_page_html:
+        zero:  "Aucune %{model} trouvé"
```
Comment (guguso) [inline lib/will_paginate/locale/fr.yml]: `Aucun` ?

**Comment by mislav** [comment]
Thanks for the contribution, but I've decided not to store translations for other languages in this library. The reason behind my decision is that I would have a hard time maintaining those translations in the long run, especially if I needed to add new keys. This library will only ship with English translation as an example of how anyone can translate will_paginate themselves in their own apps.

**Pull request closed without merging**